src/Bicep.Decompiler/BicepDecompiler.cs

```diff
@@ -101,6 +103,24 @@ public async Task<DecompileResult> Decompile(Uri entryJsonUri, Uri entryBicepUri
             PrintFiles(workspace));
     }
 
+    public string? DecompileJsonValue(string jsonInput, DecompileOptions? options = null)
+    {
+        var workspace = new Workspace();
+        var decompileQueue = new Queue<(Uri, Uri)>();
```

Comment:
nit: This is unused. Remove?

Reply:
c199856beefee9cd12af52e8f009652df7f47183